Library/Homebrew/cmd/list.rb

```diff
@@ -1,3 +1,24 @@
+#:  * `ls`, `list` [`--full-name`]:
+#:    List all installed formulae. If `--full-name` is passed, print formulae with
+#:    full-qualified names.
+#:
+#:  * `ls`, `list` `--unbrewed`:
+#:    List all files in the Homebrew prefix not installed by Homebrew.
+#:
+#:  * `ls`, `list` [`--versions` [`--multiple`]] [`--pinned`] [<formulae>]:
+#:    List the installed files for <formulae>. Combined with `--verbose`, recursively
+#:    list the contents of all subdirectories in each <formula>'s keg.
+#:
+#:    If `--versions` is passed, show the version number for installed formulae,
+#:    or only the specified formulae if <formulae> are given. With `--multiple`,
+#:    only show formulae with multiple versions installed.
+#:
+#:    If `--pinned` is passed, show the versions of pinned formulae, or only the
+#:    specified (pinned) formulae if <formulae> are given.
+#:    See also `pin`, `unpin`.
+#:  * `list`:
+#:    List all installed formulae.
```

Comment:
This line and the line above seem to be misplaced and should be removed. Where did they come from?